### DISCUSSION TITLE
Successful --onefile build

### DISCUSSION THREAD

**Discussion by jheim5** [TomSchimansky/CustomTkinter#154] [General]
Hi, I just wanted to mention that I was able to package via pyinstaller using the --onefile method. I only had to follow the instructions to --add-data for the site_packages/customtkinter.

Not sure if this justifies an update to the docs or means it would apply to everyone.
Happy to share build details if it would help. Just let me know what you would need. 

Awesome package, thank you!

**Comment by felipetesc** [comment]
I would like to know the details, so I can add to the alternative doc. Can You post it ? Thanks.

**Comment by jheim5** [comment]
It looks like all I had to do was add the path to where I stored the package in my venv. 
```
pyinstaller --icon=media/image.png \
my_app.py \
--onefile \
--name "MyApp" \
--add-data "media/*:media" \
--add-data "/home/user/.local/share/virtualenvs/exe_project-NDfeyzux/lib/python3.8/site-packages/customtkinter:customtkinter/" \
```
Basically the above worked for me. Not sure if there are additional details I can provide. 
Best!

**Comment by B16f00t** [comment]
I have also been able to create a one_file without problems, I think the wiki should be updated.
